redbeams-api/src/main/java/com/sequenceiq/redbeams/api/model/describe/DescribeRedbeamsResponse.java

```diff
@@ -0,0 +1,71 @@
+package com.sequenceiq.redbeams.api.model.describe;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.sequenceiq.redbeams.api.model.common.Status;
+import com.sequenceiq.redbeams.doc.ModelDescriptions;
+
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+
+import javax.validation.constraints.NotNull;
+
+@ApiModel("DescribeRedbeamsV1Response")
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class DescribeRedbeamsResponse {
+    @NotNull
+    @ApiModelProperty(value = ModelDescriptions.ENVIRONMENT_ID, required = true)
+    private String environmentId;
+
+    @NotNull
+    @ApiModelProperty(value = ModelDescriptions.ExternalDatabaseServer.NAME, required = true)
+    private String name;
+
+    @NotNull
+    private String crn;
```

Comment:
The emerging standard for this field seems to be "resourceCrn" rather than just "crn".